app/main.py

```diff
@@ -54,7 +54,7 @@ def create_app() -> FastAPI:
 
 This service is based on:
 
-- **oteapi-core**: v0.0.3
+- **oteapi-core**: v0.0.5
```

Comment:
This looks like a comment that is easy to forget to update. Should we remove it? The needed version of oteapi-core can be found in requirements.txt

Reply:
Sure. Otherwise we can automate it, e.g., using `invoke` - but perhaps that's overkill?

Reply:
See https://github.com/EMMC-ASBL/oteapi-services/pull/39#issuecomment-1032314133